Makefile

```diff
@@ -31,7 +33,11 @@ build-solidity-types: ## generate Solidity typings
 .PHONY: build-yarn
 build-yarn: ## build yarn packages
 build-yarn: build-solidity-types build-cargo
+ifeq ($(package),)
```

Comment:
please document the parameter in the docs line of the target

Reply:
changed

Reply:
I don't see it.

Reply:
Still not documented.